script.js

```diff
@@ -1,3 +1,7 @@
+// const { default: validation } = require("ajv/dist/vocabularies/validation");
+
+// const { message } = require("statuses");
+
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)

Reply:
resoleved and thanks